base/src/main/java/com/smartdevicelink/proxy/rpc/BodyInformation.java

```diff
@@ -118,38 +124,40 @@
  * 			<td>true</td>
  * 			<td>References signal "DrStatRr_B_Actl".</td>
  * 			<td>
- * 				@since SmartDeviceLink 2.0.0
- * 				@property-deprecated in SmartDeviceLink 7.1.0
+ * 				SmartDeviceLink 2.0.0
+ * 			 <br>
+ * 			 <br>
+ * 				Deprecated in SmartDeviceLink 7.1.0
  * 			</td>
  * 		</tr>
  * 		<tr>
  * 			<td>doorStatuses</td>
  * 			<td>List<DoorStatus></td>
- * 			<td>Provides status for doors if Ajar/Closed/Locked</td>
- * 			<td>N</td>
- * 			<td>{"array_min_size": 0, "array_max_size": 100}</td>
- * 			<td>
- * 				@since SmartDeviceLink 7.1.0
+ * 			 <td>False</td>
+ * 			<td>Provides status for doors if Ajar/Closed/Locked
+ * 			{"array_min_size": 0, "array_max_size": 100}</td>
```

Comment:
```suggestion
 * 			<td>False</td>
 * 			<td>
 *			  Provides status for doors if Ajar/Closed/Locked
 * 			  {"array_min_size": 0, "array_max_size": 100}
 * 			</td>
```

In addition to the spacing, is there a reason we're replacing "N" with "False"?